deploy/vagrant/create

```diff
@@ -65,6 +66,7 @@ if [[ "$PROVIDER" == "aws" && "$IS_SPOT" == 0 ]]; then
 else
  FULL_PROVIDER_NAME=$PROVIDER
  [[ "$FULL_PROVIDER_NAME" == "vb" ]] && FULL_PROVIDER_NAME=virtualbox
+ ./vbox/buildbox.sh
```

Comment:
Maybe put this line above like `[[ "$FULL_PROVIDER_NAME" == "vb" ]] && ./vbox/buildbox.sh && FULL_PROVIDER_NAME=virtualbox` since we only need to buildbox when provider is virtualbox.